app/models/review.rb

```diff
@@ -0,0 +1,13 @@
+class Review < ApplicationRecord
```

Comment:
please use https://github.com/ctran/annotate_models to document the model's attributes